imports/subscriptions/SearchSubscriber.ts

```diff
@@ -0,0 +1,54 @@
+import { Meteor } from 'meteor/meteor'
+import { Collections } from '../models/Collections'
+import { useEffect } from 'react'
+import { atom, useAtom } from 'jotai'
+import { IBoard } from '../excalidraw/types'
+import { globalEventEmitter } from '../utils'
+import { Services } from '../services/client'
+
+export enum SearchEvents {
+  SearchChanged = 'SearchEvents_SearchChanged'
+}
+export const searchAtom = atom<IBoard[]>([])
+
+export function SearchChanged() {
+  const [boards, setBoards] = useAtom(searchAtom)
+  useEffect(() => {
+    const onSearchChanged = (boards: IBoard[]) => setBoards(boards)
+    globalEventEmitter.on(SearchEvents.SearchChanged, onSearchChanged)
+    SearchSubscriber.subscribe()
+    return () => {
+      globalEventEmitter.off(SearchEvents.SearchChanged, onSearchChanged)
+    }
+  }, [])
+  return boards
+}
+
+class SearchSubscriberClass {
```

Comment:
maybe for search results, we can use `searchBoards` in BoardService. we don't need to introduce `SearchSubscriber` to make the search results "reactive". it's too heavy.

we can call `Services.get('boards').searchBoards` when use typing search keywords in input box. I think this way is more simple.